src/Main.java

```diff
@@ -1,4 +1,44 @@
 public class Main {
     public static void main(String[] args) {
+        //System.out.println("Задача 1");
+        int[] array1 = new int[]{1, 2, 3};
```

Comment:
этот способ равносилен такому
```suggestion
        int[] array1 = {1, 2, 3};
```
Требовалось создать массив указанной длины и заполнить его уже после создания, как в примере показано было